x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/custom_timeline_data_grid_body.tsx

```diff
@@ -122,34 +160,48 @@ const CustomDataGridSingleRow = memo(function CustomDataGridSingleRow(
   );
   const eventTypeRowClassName = useGetEventTypeRowClassName(rowData.ecs);
 
+  const isRowIntersecting =
```

Comment:
I'm not sure if the browser is catching this as quickly as it should. I'm seeing  behavior where all the rows disappear on scroll after making it 100 rows. The below captures it a bit after it started happening to me

https://github.com/elastic/kibana/assets/17211684/228b5d03-ef4f-4a1c-806e-aacb55fe83d0